includes/widgets/image.php

```diff
@@ -600,6 +600,37 @@ private function get_caption( $settings ) {
 		return $caption;
 	}
 
+	public function set_svg_size( $image_data, $attachment_id, $size ) {
```

Comment:
What about other widgets with a `media` control?

Reply:
Fixed - Added support in Image Box, Image Carousel and Image Gallery,